tools/clang/lib/SPIRV/DeclResultIdMapper.cpp

```diff
@@ -203,9 +203,9 @@ SpirvEvalInfo DeclResultIdMapper::getDeclResultId(const NamedDecl *decl) {
           cast<VarDecl>(decl)->getType(),
           // We need to set decorateLayout here to avoid creating SPIR-V
           // instructions for the current type without decorations.
-          // According to the Vulkan spec, cbuffer should follow standrad
-          // uniform buffer layout, which GLSL std140 rules statisfies.
-          LayoutRule::GLSLStd140);
+          // According to the Vulkan spec, cbuffer should follow GLSL std140
```

Comment:
Can remove these two lines of comments since we are fetching the layout rule from `info`.

Reply:
Done